static_routing.md: added example with figure (#117)
bisdn/bisdn.github.io: network_configuration/static_routing.md

Adding the -4/6 argument to the call allows to show only the desired routes/ addresses by IP protocol.

For ‘systemd-networkd’ the configuration file is done the same way.

## Static route example

> Here we will give a simple example of using static routes on four nodes. Two

Reviewer: ```suggestion
Here we show a simple example of using static routes on four nodes. Two
```

> switches and two server, as shown on the figure below.

Reviewer 2: ```suggestion
switches and two servers, as shown on the figure below.
```

#            |                       |                       |                     |
#            |                       |                       |                     |
#            |                       +-----------------------+                     |
#            |                                                                     |
#            |                                                                     |
#            |                                                                     |
#  ++-----------------------------+                             +----------+-------+----------++
#  ||      eno2        |          |                             |          |      eno2        ||
#  |+------------------+          |                             |          +------------------+|
#  |       10.0.1.2/24            |                             |                 10.0.2.2/24  |
#  |                              |                             |                              |
#  | server-1                     |                             | server-2                     |
#  +------------------------------+                             +------------------------------+
#

> 

Reviewer 2: It might be worthwhile to add some textual explanation here on assumptions, i.e. the servers are using systemd-networkd for network configuration as well, and the network interfaces connected to the switches is called eno2.

Author: Added a sentence explaining the figure

# Setup switch-1

> Set ``port2`` up and add the ip address 10.0.1.1/24.

Reviewer 2: ```suggestion
Set ``port2`` up and add the IP address 10.0.1.1/24.
```

> This sets ``port2`` up and adds the ip address 10.0.1.1/24.

Reviewer: ```suggestion
Set ``port2`` up and add the ip address 10.0.1.1/24.
```

`switch-1 /etc/systemd/network/30-port2.network`
```
[Match]
Name=port2
[Network]
Address=10.0.1.1/24
```

> Set ``port54`` up, add ip address 10.0.3.1 and add a route to the

Reviewer 2: ```suggestion
Set ``port54`` up, add IP address 10.0.3.1 and add a route to the
```

> This sets ``port54`` up, adds ip address 10.0.3.1 and adds a route to the

Reviewer: ```suggestion
Set ``port54`` up, add ip address 10.0.3.1 and add a route to the
```

> 10.0.2.0/24 subnet (which is on ``server-2``) via 10.0.3.2

Reviewer: ```suggestion
10.0.2.0/24 subnet (which is on ``server-2``) via 10.0.3.2.
```

`switch-1 /etc/systemd/network/30-port54.network`
```
[Match]
Name=port54
[Network]
Address=10.0.3.1/24
[Route]
Destination=10.0.2.0/24
Gateway=10.0.3.2
```

# Setup switch-2

> Set ``port2`` up and add ip address 10.0.2.1/24.

Reviewer 2: ```suggestion
Set ``port2`` up and add IP address 10.0.2.1/24.
```

`switch-2 /etc/systemd/network/30-port2.network`
```
[Match]
Name=port2
[Network]
Address=10.0.2.1/24
```

> Set ``port54`` up, add ip address 10.0.3.2 and add a route to the

Reviewer 2: ```suggestion
Set ``port54`` up, add IP address 10.0.3.2 and add a route to the
```

> This sets ``port54`` up, adds ip address 10.0.3.2 and adds a route to the

Reviewer: ```suggestion
Set ``port54`` up, add ip address 10.0.3.2 and add a route to the
```

> 10.0.1.0/24 subnet (which is on ``sever-1``) via 10.0.3.1

Reviewer: ```suggestion
10.0.1.0/24 subnet (which is on ``sever-1``) via 10.0.3.1.
```

`switch-2 /etc/systemd/network/30-port54.network`
```
[Match]
Name=port54
[Network]
Address=10.0.3.2/24
[Route]
Destination=10.0.1.0/24
Gateway=10.0.3.1
```

# Setup server-1

> Add ip address 10.0.1.2/24 to ``eno2`` and a route to the subnet

Reviewer 2: ```suggestion
Add IP address 10.0.1.2/24 to ``eno2`` and a route to the subnet
```

> This adds ip address 10.0.1.2/24 to ``eno2`` and a route to the subnet

Reviewer: ```suggestion
Add ip address 10.0.1.2/24 to ``eno2`` and a route to the subnet
```

> 10.0.2.0/24 (which is on ``server-2``) via 10.0.1.1

Reviewer: ```suggestion
10.0.2.0/24 (which is on ``server-2``) via 10.0.1.1.
```

`server-1 /etc/systemd/network/30-eno2.network`
```
[Match]
Name=eno2
[Network]
Address=10.0.1.2/24
[Route]
Destination=10.0.2.0/24
Gateway=10.0.1.1
```

# Setup server-2

> Add ip address 10.0.2.2/24 to ``eno2`` and a route to the subnet

Reviewer 2: ```suggestion
Add IP address 10.0.2.2/24 to ``eno2`` and a route to the subnet
```

> This adds ip address 10.0.2.2/24 to ``eno2`` and a route to the subnet

Reviewer: ```suggestion
Add ip address 10.0.2.2/24 to ``eno2`` and a route to the subnet
```

> 10.0.1.0/24 (which is on ``server-1``) via 10.0.2.1

Reviewer: ```suggestion
10.0.1.0/24 (which is on ``server-1``) via 10.0.2.1.
```

`server-2 /etc/systemd/network/30-eno2.network`
```
[Match]
Name=eno2
[Network]
Address=10.0.2.2/24
[Route]
Destination=10.0.1.0/24
Gateway=10.0.2.1
```

> Restart systemd-networkd or reboot the switches to apply network configuration.

Reviewer 2: What about the  servers?

Author: Completely true, forgot that the routes to 10.0.x.2/24 make no sense without the server config.

Author: fixed

Adding the -4/6 argument to the call allows to show only the desired routes/ addresses by IP protocol.

For ‘systemd-networkd’ the configuration file is done the same way.

## Static route example

Here we show a simple example of using systemd-networkd to configure static

> routes on two switches and two servers.

Reviewer 2: None of the other sections in this file use manual line breaks, so I don't think you should add them.
